test/python/transpiler/test_dynamical_decoupling.py

```diff
@@ -1031,6 +1031,28 @@ def test_respect_target_instruction_constraints(self):
         expected.delay(200, [0])
         self.assertEqual(expected, scheduled)
 
+    def test_paramaterized_global_phase(self):
+        """Test paramaterized global phase in DD circuit.
+        See:https://github.com/Qiskit/qiskit-terra/issues/10569
+        """
+        dd_sequence = [XGate(), YGate()] * 2
+        qc = QuantumCircuit(1, 1)
+        qc.h(0)
+        qc.delay(1700, 0)
+        qc.y(0)
+        qc.global_phase = Parameter("a")
+        pm = PassManager(
+            [
+                ALAPScheduleAnalysis(self.durations),
+                PadDynamicalDecoupling(self.durations, dd_sequence),
+            ]
+        )
+        try:
```

Comment:
I'd suggest replacing this block with an assertion. Either just on `pm.run(qc)`, or perhaps on `qc.global_phase` equal to that of the output circuit

Reply:
I have replaced it with following code block:
```
self.assertEqual(qc.global_phase + 3.14159265358979, pm.run(qc).global_phase)
```
But getting following error while testing:
```
AssertionError: ParameterExpression(a + 3.14159265358979) != ParameterExpression(a + 3.14159265358979)
```
Can you please help me in debugging this .